client-src/elements/chromedash-userlist.ts

```diff
@@ -93,21 +92,31 @@ class ChromedashUserlist extends LitElement {
   // TODO(jrobbins): Change this to be a JSON API call via csClient.
   async ajaxSubmit(e) {
     e.preventDefault();
-    const formEl = this.shadowRoot.querySelector('form');
+    const formEl: HTMLFormElement | null =
+      this.renderRoot.querySelector('form');
 
-    if (formEl.checkValidity()) {
-      const email = formEl.querySelector('input[name="email"]').value;
-      const isAdmin = formEl.querySelector('input[name="is_admin"]').checked;
-      const isSiteEditor = formEl.querySelector(
+    if (formEl && formEl.checkValidity()) {
+      const emailInput: HTMLInputElement | null = formEl.querySelector(
+        'input[name="email"]'
```

Comment:
`querySelector()`'s typing has an overload that lets you treat it as any type, even if the selector can't possibly return an element of that type. As a matter of taste, I'd use `as HTMLInputElement` in that case, to emphasize to the reader that you're asserting a type, not writing a type that the compiler will check.